riscv-executor/src/lib.rs

```diff
@@ -1138,6 +1138,12 @@ impl<'a, 'b, F: FieldElement> Executor<'a, 'b, F> {
 
                 vec![]
             }
+            "commit_public" => {
```

Comment:
We should probably return these at some point when the clients of riscv-executor need it.